addons/website/views/res_config_settings_views.xml

```diff
@@ -120,6 +124,26 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box"
+                            id="website_login_documents"
+                            title=" To send invitations in B2B mode, open a contact or select several ones in list view and click on 'Portal Access Management' option in the dropdown menu *Action*.">
+                            <div class="o_setting_left_pane">
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="auth_signup_uninvited"/>
+                                <div class="text-muted">
+                                    Let your customers log in to see their documents
+                                </div>
+                                <div class="mt8">
+                                    <field name="auth_signup_uninvited" class="o_light_label" widget="radio" options="{'horizontal': true}" required="True"/>
```

Comment:
This can't be `required="True"`, it makes the settings impossible to save (from every module, not only website) in multi company cases
In such cases, there is no website selected.
```xml
attrs="{'required': [('website_id', '!=', False)]}" 
```

https://watch.screencastify.com/v/1nfa9qqMJm3MZGQkADid

@pierre-pde did you test multi company cases? As the website settings is totally unusual (the whole setting screen are just related field to the selected website), multi website and multi company cases are always something to consider there

Reply:
Oh right.

 I will do a fix for this straight away.

Reply:
Hi @rdeodoo , No I haven't tested in multicompany, thanks for the catch.

Reply:
Follow-up here: https://github.com/odoo/odoo/pull/89828

Reply:
Thanks for the prompt reaction.
I added a few comments and r- so we have time to finish the discussion up there :+1: